hermes_cms/hermes_cms/views/admin.py

```diff
@@ -18,9 +22,57 @@ def index():
     return Response(response=lookup.get_template('index.html').render(), status=200)
 
 
-@route.route('/logout', methods=['GET'])
-def logout():
-    if Auth.delete_session():
-        return redirect('/')
+@route.route('/user', methods=['GET'])
+def user_get():
```

Comment:
Is it possible to get anything other than 200 OK response ? We need to use apiary to design this RESTful interfaces.